### ISSUE TITLE
What are the labels training the depth estimator

### ISSUE THREAD

**Issue by hellojialee** [vegesm/depthpose#3]
Thank you for your great work!

Do you train (fine-tune) the depth network only with the depth values at the keypoint locations predicted by the 2D pose estimator? It puzzle me that only a number of pixels in the depth map are supervised during training can be successful (the training labels seem sparse).

**Comment by vegesm** [comment]
The network is pretrained on a dataset which has dense labels. Then it is
finetuned for the pose estimation task using sparse labels only.

Jia Li ***@***.***> ezt írta (időpont: 2021. dec. 21., Ke
5:54):

**Comment by hellojialee** [comment]
Thank you! Thus, the depth network was fine-tuned by the values of Z coordinates of the keypoints.
